graphql_compiler/compiler/ir_lowering_match/__init__.py

```diff
@@ -17,12 +19,13 @@
                                  lower_context_field_expressions, prune_non_existent_outputs)
 from ..match_query import convert_to_match_query
 from ..workarounds import orientdb_class_with_while, orientdb_eval_scheduling
-
+from .utils import _construct_where_filter_predicate
```

Comment:
Underscored functions should be private and not imported (except _maybe_ in tests). Remove the underscore from the function name.